composer.json

```diff
@@ -6,33 +6,25 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.2",
-        "drupal/core": "^8.7",
-        "drupal/language_selection_page": "2.3",
+        "php": ">=7.3",
+        "drupal/core": "^8.9 || ^9",
+        "drupal/language_selection_page": "^2.7",
```

Comment:
This won't work without this patch: https://git.drupalcode.org/project/language_selection_page/-/merge_requests/1/diffs

So let's add this please and it needs to be reviewed as I made it.